deploy/deploy

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
 # Deploy Script for Frontend and Backend
 ssh -tt ubuntu@quadcore.news 'bash -s' << 'DEPLOY'
+sudo systemctl daemon-reload
 sudo systemctl stop nginx
 sudo systemctl stop quadcore
 
 cd /var/www
 sudo rm -rf quadcore.news
 
-git clone https://github.com/kmu-sv/quadcore-serving.git
+sudo git clone https://github.com/kmu-sv/quadcore-serving.git
```

Comment:
why do we need a sudo's here? where are we running this script?

Reply:
In `/var/www`, where needs enough permission to write some files. If I except `sudo` on that command, it'll occur crash with `Permission Denied`.